ResourceStatusSystem/Agent/RucioRSSAgent.py

```diff
@@ -0,0 +1,69 @@
+""" :mod: RucioRSSAgent
+
+    Agent that synchronizes Rucio and Dirac
+"""
+
+# # imports
+from DIRAC import S_OK, S_ERROR
+from DIRAC import gConfig, gLogger
+from DIRAC.Core.Base.AgentModule import AgentModule
+from DIRAC.Core.DISET.RPCClient import RPCClient
+from DIRAC.ResourceStatusSystem.Client.ResourceStatus import ResourceStatus
+
+from rucio.client import Client
+
+__RCSID__ = "Id$"
+
+class RucioRSSAgent(AgentModule):
+  """
+  .. class:: RucioRSSAgent
+
+  Agent that synchronizes Rucio and Dirac
+  """
+
+  def initialize(self):
+    """ agent's initalisation
+
+    :param self: self reference
+    """
+    gLogger.info("Starting RucioRSSAgent")
+    return S_OK()
+
+  def execute(self):
+    """ execution in one agent's cycle
+
+    :param self: self reference
+    """
+    rSS = ResourceStatus()
+    client = Client()
+    try: 
+      for rse in client.list_rses():
+        thisSe = rse['rse']
+        gLogger.info("Running on %s" % thisSe)
+        resStatus =  rSS.getElementStatus(thisSe, "StorageElement")
+        dictSe = client.get_rse(thisSe)
+        if resStatus['OK']:
+          seAccessValue = resStatus['Value'][thisSe]
+          availabilityRead = True if seAccessValue['ReadAccess'] == 'Active' else False
+          availabilityWrite = True if seAccessValue['WriteAccess'] == 'Active' else False
+          availabilityDelete = True if seAccessValue['RemoveAccess'] == 'Active' else False
```

Comment:
RSS has actually 4 states, you states, you can read about them in https://dirac.readthedocs.io/en/latest/AdministratorGuide/Systems/ResourceStatus/introduction.html#state

So I think you can do 

```suggestion
          availabilityRead = True if seAccessValue['ReadAccess'] in ('Active', 'Degraded') else False
          availabilityWrite = True if seAccessValue['WriteAccess'] in ('Active', 'Degraded') else False
          availabilityDelete = True if seAccessValue['RemoveAccess'] in ('Active', 'Degraded') else False
```

Reply:
Done